packages/core/src/external/commonwell/document/document-downloader-local.ts

```diff
@@ -245,4 +257,22 @@ export class DocumentDownloaderLocal extends DocumentDownloader {
       throw new MetriportError(`CW - Error downloading document`, error, additionalInfo);
     }
   }
+
+  protected async updateContentTypeInS3(
+    bucket: string,
+    key: string,
+    newContentType: string
+  ): Promise<void> {
+    const copySource = encodeURIComponent(bucket + "/" + key);
```

Comment:
This is good! 😍 

I suggest we move this to a function on `s3.ts` and also add `encodeURIComponent` to `createS3FileName()`.